examples/cms-sitecore/README.md

```diff
@@ -0,0 +1,122 @@
+# A statically generated blog example using Next.js and Sitecore Experience Edge (Edge for Content Hub).
+
+This example showcases Next.js's [Static Generation](https://nextjs.org/docs/basic-features/pages) feature using [Sitecore Experience Edge for Content Hub](https://docs.stylelabs.com/contenthub/4.1.x/content/user-documentation/experience-edge/caas-intro.html) as the data source.
+
+# Live Demo
+[https://nextjs-blog-sitecore.vercel.app/](https://nextjs-blog-sitecore.vercel.app/)
```

Comment:
Could we deploy this? 😄

Reply:
Yes you can. Only if you have an up and running Sitecore Content Hub sandbox.